src/lib/constants.ts

```diff
@@ -22,3 +22,6 @@ export const SM_ROLE_HASHES = [
   Role.CLAIM_OPERATOR_ROLE,
   Role.STAKE_OPERATOR_ROLE,
 ].map(getRole);
+
+// DIP_6_IPFS_HASH taken from the dydxfoundation/dip/content/ipfs-dip/DIP-6-Ipfs-hashes file
```

Comment:
Mind adding the whole link so people can copy paste into their browers? Ex: https://github.com/dydxfoundation/dip/blob/master/content/ipfs-dips/DIP-6-Ipfs-hashes.json#L4

Also, add trailing `;`.